Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2239,7 +2226,6 @@ - (void)ensureScrollDownButton
     OWSAssert([NSThread isMainThread]);
 
     BOOL shouldShowScrollDownButton = NO;
-    NSUInteger numberOfMessages = [self.messageMappings numberOfItemsInSection:0];
```

Comment:
We want to treat the "view items" as the single source of truth around the collection view contents and consult the `messageMappings` only when updating the "view items".